docs/troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+# Troubleshooting
+
+## PR-traffic-light failures:
+#### Operator changes detected with ci changes and 'allow/ci-changes' is not set
+You are unintentionally editing a file which is out of your responsibility, please rebase.
+
+#### Changes in both 'community-operators' and 'upstream-community-operators' dirs
+Every operator must have separate PR. Separate for Kubernetes (upstream) and separate for Openshift (community). It helps more precise testing and also granular revert if needed. Sometimes just rebase is needed.
+
+#### Multiple operators are changed
+Multiple operators in the same stream are updated. This is not allowed. It could be a result of not rebased PR. Please rebase or create separate PR for every operator.
+
+## Kiwi test failures:
```

Comment:
Package metadata test (kiwi) instead of kiwi. This is proposed by @dmesser in https://hotovo.jira.com/browse/RHO-661